components/hab/static/template_init

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
```

Comment:
I think you don't want to do it at all, actually - the "standard" should be to just use the auto-generated run file the plan gives you if you fill in the `pkg_svc_run` field.